### ISSUE TITLE
compile_state.tau_mem is not set for LeakyIntegrate in EProp Compiler

### ISSUE THREAD

**Issue by FabianSchubert** [genn-team/ml_genn#97]
Using `LeakyIntegrate` as an output seems to cause an assertion error `assert self._tau_mem is not None` when trying to compile with `EPropCompiler`. My guess is that
https://github.com/genn-team/ml_genn/blob/1a27df5c561f40a70783a4b43ff76f0b54e22dbd/ml_genn/ml_genn/compilers/eprop_compiler.py#L419-L420

should just include `LeakyIntegrate` so that `compile_state.tau_mem` is set?

**Comment by neworderofjamie** [comment]
That branch is only for hidden neurons - do you definitely have a suitable readout set for your ``LeakyIntegrate`` output?

**Comment by FabianSchubert** [comment]
Yes, I directly connected a spiking input population to the `LeakyIntegrate` output, so indeed there were no other neurons with tau. It was just a linear regression toy model that I was trying to set up, so this error probably doesn't have any practical relevance.
